scripts/install.sh

```diff
@@ -51,38 +51,65 @@ else
     echo -e "${ORANGE}Please look up how to install Python 3.9 or later for your specific operating system.${NC}"
     exit 1
 fi
-
 #
-# Install needed system packages
+# Get os package manager
 #
-# Run apt-get update
-echo -e "${GREEN}Updating apt-get cache...${NC}"
-sudo apt-get update || exit 1
+if [[ -f /etc/os-release ]]; then
+    source /etc/os-release
+    if [[ "$ID_LIKE" == *"debian"* ]]; then
```

Comment:
Turns out on Debian itself, ID_LIKE doesn't exist, only ID does, which is "debian". So you should check for either ID_LIKE like you do here, or `"$ID" == *"debian"*`

![image](https://github.com/Donkie/Spoolman/assets/2332094/79770192-17aa-4e44-be47-1b59a00f614a)

Reply:
I check for both ID and ID_LIKE variables, this should do the trick for you.